ios/RCTToast/Toast.m

```diff
@@ -1,6 +1,6 @@
 #import <UIKit/UIKit.h>
-#import "RCTLog.h"
-#import "RCTBridgeModule.h"
+#import "React/RCTLog.h"
```

Comment:
If you want retrocompatibility with previous versions, this might help : 
```
if __has_include(<React/RCTLog.h>)
  #import <React/RCTLog.h>
  #import <React/RCTBridgeModule.h>
#else
  #import "RCTLog.h"
  #import "RCTBridgeModule.h"
#endif
```